.gitignore

```diff
@@ -1,12 +1,11 @@
-/Cargo.lock
-/target
-**/*.rs.bk
-src/__pycache__/*
-.gocache
-maipo/
-*.pyc
-*.pyo
+# generated by Python
+__pycache__/
+
+# generated by `make shellcheck`
 *.shellchecked
+
+# generated by `pytest`
 .coverage
-.idea
+
+# generated by `make all`
 bin/
```

Comment:
Should these two patterns be anchored to the root directory, e.g. `/bin/`?

Reply:
Done!